### ISSUE TITLE
Use ISABELLE_IDENTIFIER instead of ISABELLE_VERSION

### ISSUE THREAD

**Issue by dominique-unruh** [larsrh/multi-isabelle#1]
I suggest to replace the following line
https://github.com/larsrh/multi-isabelle/blob/f4b01febd9abd2fa13c03994574ae0c5b49b1fb8/src/main/isabelle/ml_cond.ML#L14
by
```
val isabelle_version = getenv_strict "ISABELLE_IDENTIFIER" |> unprefix "Isabelle"
```

The latter uses an environment variable that is set by Isabelle, and thus works even if Multi_Isabelle.thy is loaded in jEdit.

**Comment by larsrh** [comment]
Fixed in 40a28b3d5a0ecba6da1a56d976b51592e4cd9e9e. You can override the `multi-isabelle` version to `0.1.4` (I borked the `0.1.3` release) in your local build:

    libraryDependencies += "info.hupel" % "multi-isabelle" % "0.1.4"

**Comment by dominique-unruh** [comment]
It works fine. Thanks.
